src/components/Pages/Guide/GuideMenu.js

```diff
@@ -7,8 +7,7 @@ import { isMobileOrTablet } from 'js/helpers/reactMediaQueries';
 function GuideMenuLink({ title, anchorTag, isHeading, isCurrentSection }) {
   // Each GuideSectionWrapper has an id={this.props.anchorTag}
   function goToSection(e) {
-    e.preventDefault();
-    window.location.href = `#${anchorTag}`;
+    document.getElementById(anchorTag).scrollIntoView(true);
```

Comment:
Here is where an element of where the issue was coming from. 
-
At this point, it seems the body of the guide hadn't been updated completely to the DOM, where the scroll-to toward the `id` of the anchor is broken mid path. by targeting the element directly, it'll only start the operation when it's mounted. Rather, then twice through the update of the url.... if that makes sense.

Reply:
Awesome!